project/plugins.sbt

```diff
@@ -19,8 +19,9 @@ addSbtPlugin("com.typesafe.sbt" % "sbt-gzip" % "1.0.2")
 addSbtPlugin("org.irundaia.sbt" % "sbt-sassify" % "1.4.12")
 
 // S3 based SBT resolver
-resolvers += Resolver.jcenterRepo
-addSbtPlugin("ohnosequences" % "sbt-s3-resolver" % "0.19.0")
+
+resolvers += "FrugalMechanic Snapshots" at "s3://maven.frugalmechanic.com/snapshots"
```

Comment:
no need for a resolver, they publish to Maven central

Reply:
I thought we were publishing to S3 ?

Reply:
Ok great, once again copied code will fix